Gems/Atom/RHI/Code/Include/Atom/RHI/RayTracingBufferPools.h

```diff
@@ -43,6 +44,7 @@ namespace AZ::RHI
 
         virtual RHI::BufferBindFlags GetShaderTableBufferBindFlags() const { return RHI::BufferBindFlags::ShaderRead | RHI::BufferBindFlags::CopyRead | RHI::BufferBindFlags::RayTracingShaderTable; }
         virtual RHI::BufferBindFlags GetScratchBufferBindFlags() const { return RHI::BufferBindFlags::ShaderReadWrite | RHI::BufferBindFlags::RayTracingScratchBuffer; }
+        virtual RHI::BufferBindFlags GetAabbStagingBufferBindFlags() const { return RHI::BufferBindFlags::CopyRead; }
```

Comment:
A stylistic question, would it not be cleaner to make these functions `public` and use them for the buffers as well, since that is currently duplicated now in multiple places?

Reply:
I know that would be a bit more of a refactor, but wouldn't that be a lot cleaner and less error-prone?

Reply:
I had similar thoughts when I saw this code; also why are `m_bindFlags` in `RHI::BufferInitRequest` needed if they must be the same as for the buffer pool and there is an error if they are not?